src/structures/Contact.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const Base = require('./Base');
+
+/**
+ * Represents a Contact on WhatsApp
+ * @extends {Base}
+ */
+class Contact extends Base {
+    constructor(client, data) {
+        super(client);
+
+        if(data) this._patch(data);
+    }
+
+    _patch(data) {
+        this.id = data.id;
+        this.isBusiness = data.isBusiness;
+        this.isEnterprise = data.isBusiness;
```

Comment:
Typo here? Should be `data.isEnterprise`

Reply:
Yes ofc. My bad.

Reply:
fixed